ManatiCR/vitaminau
# VIT-79 Change telephone for cell phone.

# [VIT-79](https://manati.atlassian.net/browse/VIT-79)
Change phone to cell phone.

## Intallation
- [x] ahoy drush cim -y && ahoy drush cr

## Test
- Go to [register](http://vitaminau.docker/user/register). (You need to be log out for see the change)
- Check if the label change to cell phone.
